Download template tag json file when needed.

skiphansen opened the pull request.

nlimper commented on ESP32_AP-Flasher/wwwroot/main.js:
```diff
@@ -1419,11 +1423,13 @@ async function getTagtype(hwtype) {
 			shortlut: parseInt(jsonData.shortlut),
-			busy: false
+			busy: false,
+			usetemplate:parseInt(jsonData.usetemplate || "0",16)
```
usetemplate is defined in the json as a decimal integer, not a hex (yes, that's a bit inconvenient when creating a tagtype definition, but it's a bit too much to change it and have it backwards compatible)

nlimper commented:
Nice and quick way to download the tagtype json refered by usetemplate. But during an tagtype update it will be automatically deleted, and probably not immediately downloaded again because the object is cached. We need to fix that too.

skiphansen commented:
I'll take a look and see if I can address that issue as well.  I really don't know Javascript, I'm just faking it!
I'm also working on adding the subGhz channel to the ap card.

jjwbruijn commented:
I’ve been faking it for years, I think most of us have :) Some just hide it better than others

skiphansen commented:
I've updated the PR and I think I've fixed the update issue.  

To test it I installed the 2.52 release, changed the update Repo to skiphansen and then updated to the pr7_test release.  

/tagtypes ended up with

![image](https://github.com/jjwbruijn/OpenEPaperLink/assets/4642629/b24e824a-7b2b-461f-88cf-7d69f09cfdee)

and content generation works.

I've left the pr7_test release on my repo.

nlimper commented:
Nice solution! Simple and very efficient.

skiphansen commented:
Thank you!
